src/helpers/explore-url.ts

```diff
@@ -64,6 +75,12 @@ export const defaultAnalysisMode: AnalysisMode = AnalysisMode.Single;
 
 export const defaultHost: HostSelector = [HUMAN];
 
+const deleteSubmissionDateParams = (params: URLSearchParams) => {
```

Comment:
I think that this should be moved to `DateRangeSelector` where also the other URLSearchParams-manipulating functions for date ranges are located.